dbfit-java/build.gradle

```diff
@@ -222,3 +251,114 @@ project('netezza') {
     }
 }
 
+// Aggregated dependencies to include in the POM file when publishing.
+dependencies {
+    compile publicCoreCompileDeps
+    runtime publicHsqldbRuntimeDeps
+    compile publicOracleCompileDeps
+    compile publicMysqlCompileDeps
+    compile publicPostgresCompileDeps
+}
+
+// Remove the default artifact.
```

Comment:
After some digging into forums (perhaps I should read the docs at some point :-)):
* Looks like removing all defaults artifacts would give the same result right now: `configurations.archives.artifacts.clear()`
* If a need emerges to have non-JAR which we need to keep:
`configurations.archives.artifacts.removeAll { it.archiveTask.is jar }`